scripts/firedrake-install

```diff
@@ -142,6 +142,27 @@ Please follow the instructions at http://www.firedrakeproject.org/download.html
             print("""\nfiredrake-update must be run with Python 3, did you accidentally use Python 2?""")
     sys.exit(1)
 
+# Temporary catch for tigerlake processors with gcc/g++/gfortran 9.3
+# But only for Linux
+if platform.system() == "Linux":
+    # Get the cpu information
+    cpuinfostring = os.popen('lscpu').read()
+    # Check if it is a tigerlake processor
+    if cpuinfostring.find('i3-11') != -1 or cpuinfostring.find('i5-11') != -1 or cpuinfostring.find('i7-11') != -1 or cpuinfostring.find('i9-11') != -1:
+        # Get the compiler versions
+        gccversionstring = os.popen('gcc --version').read()
+        gccversionstring = gccversionstring[gccversionstring.find(')')+2:gccversionstring.find('\n')]
```

Comment:
Can we avoid hand-rolled version string parsing?

Reply:
`packaging.version.parse(gccversionstring)`

Reply:
That is a great package!